src/ios/getHeadersInFolder.js

```diff
@@ -1,7 +1,7 @@
 const glob = require('glob');
 const path = require('path');
 
-const GLOB_EXCLUDE_PATTERN = ['node_modules/**', 'Examples/**', 'examples/**'];
+const GLOB_EXCLUDE_PATTERN = ['node_modules/**', 'Pods/**', 'Examples/**', 'examples/**'];
```

Comment:
Kind of, he added that to `rnpm` itself that has similar GLOB. the difference is that in rnpm we look for .xcodeproj, here we look for headers so I just thought like it might be useful.